src/register/RegistrationPage.jsx

```diff
@@ -83,16 +76,7 @@ class RegistrationPage extends React.Component {
         username: '',
         password: '',
         country: '',
-        honorCode: '',
```

Comment:
Removing this would was cause failure in `handleOnChange` for honorCode. 
`[e.target.name]: targetValue,`